### PR TITLE
Add source code and source code hash to upload

### PR DESCRIPTION
**Pull request by Brewer-Slack**
I think I figured this out. This is implemented so that we will pass along String lists containing however many source codes included for each index. Tim's implementation assumes that a script can have more than one source, so we have to handle that as well. Please let me know if there are any issues with this!

### CODE REVIEW

File: src/main/java/org/yesworkflow/extract/DefaultExtractor.java [hunk 1]
```diff
@@ -149,6 +150,15 @@ public String getSkeleton() {
 	    
         return skeleton;
     }
+
+    @Override
+    public List<String> getSourceCodeList() {
```
Comment (aniehuser) [inline src/main/java/org/yesworkflow/extract/DefaultExtractor.java]: You don't use `@Override` on a method that should be implemented from an interface in Java

Comment (aniehuser) [reply src/main/java/org/yesworkflow/extract/DefaultExtractor.java]: Add `public List<String> getSourceCodeList()` to the [Extractor](https://github.com/aniehuser/yw-prototypes/blob/master/src/main/java/org/yesworkflow/extract/Extractor.java) interface

**Comment by Brewer-Slack** [comment]
The build is still erroring for me. Looks like an issue with maven.

**Comment by Brewer-Slack** [comment]
It passed through Travis without an issue, so I'm assuming that the error is on my system rather than in the code.

**Pull request merged**